google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/ConditionalRowMutation.java

```diff
@@ -136,4 +137,20 @@ public CheckAndMutateRowRequest toProto(RequestContext requestContext) {
         .setAppProfileId(requestContext.getAppProfileId())
         .build();
   }
+
+  /**
+   * Wraps the protobuf {@link CheckAndMutateRowRequest}.
+   *
+   * <p>WARNING: Please note that the table name will be overwritten by the configuration in the
+   * BigtableDataClient. The mutation must always be idempotent because it would be retried.
+   */
+  @BetaApi
+  public static ConditionalRowMutation fromProtobuf(@Nonnull CheckAndMutateRowRequest request) {
```

Comment:
Here and everywhere else, please stay consistent with existing `fromProto` methods